utils.py

```diff
@@ -0,0 +1,195 @@
+
+
+def str_to_felt(text):
+    """Return a field element from a given string."""
+    b_text = bytes(text, "ascii")
+    return int.from_bytes(b_text, "big")
+
+
+def felt_to_str(felt):
+    """Return a string from a given field element."""
+    b_felt = felt.to_bytes(31, "big")
+    return b_felt.decode()
+
+
+def to_uint(a):
+    """Return uint256-ish tuple from value."""
+    return (a & ((1 << 128) - 1), a >> 128)
+
+
+def from_uint(uint):
+    """Return value from uint256-ish tuple."""
+    return uint[0] + (uint[1] << 128)
+
+
+def add_uint(a, b):
+    """Return the sum of two uint256-ish tuples."""
+    a = from_uint(a)
+    b = from_uint(b)
+    c = a + b
+    return to_uint(c)
+
+
+def sub_uint(a, b):
+    """Return the difference of two uint256-ish tuples."""
+    a = from_uint(a)
+    b = from_uint(b)
+    c = a - b
+    return to_uint(c)
+
+
+def mul_uint(a, b):
+    """Return the product of two uint256-ish tuples."""
+    a = from_uint(a)
+    b = from_uint(b)
+    c = a * b
+    return to_uint(c)
+
+
+def div_rem_uint(a, b):
+    """Return the quotient and remainder of two uint256-ish tuples."""
+    a = from_uint(a)
+    b = from_uint(b)
+    c = math.trunc(a / b)
+    m = a % b
+    return (to_uint(c), to_uint(m))
+
+
+async def assert_revert(fun, reverted_with=None):
+    """Raise if passed function does not revert."""
+    try:
+        await fun
+        raise AssertionError("Transaction did not revert")
+    except StarkException as err:
+        _, error = err.args
+        if reverted_with is not None:
+            assert reverted_with in error["message"]
+
+
+async def assert_revert_entry_point(fun, invalid_selector):
+    """Raise is passed function does not revert with invalid selector."""
+    selector_hex = hex(get_selector_from_name(invalid_selector))
+    entry_point_msg = f"Entry point {selector_hex} not found in contract"
+
+    await assert_revert(fun, entry_point_msg)
+
+
+def assert_event_emitted(tx_exec_info, from_address, name, data):
+    """Raise if event and event items do not match."""
+    assert (
+        Event(
+            from_address=from_address,
+            keys=[get_selector_from_name(name)],
+            data=data,
+        )
+        in tx_exec_info.raw_events
+    )
+
+
+def _get_path_from_name(name):
+    """Return the contract path by contract name."""
+    dirs = ["src", "tests/mocks"]
+    for dir in dirs:
+        for (dirpath, _, filenames) in os.walk(dir):
+            for file in filenames:
+                if file == f"{name}.cairo":
+                    return os.path.join(dirpath, file)
+
+    raise FileNotFoundError(f"Cannot find '{name}'.")
+
+
+def get_contract_class(contract, is_path=False):
+    """Return the contract class from the contract name or path."""
+    if is_path:
+        path = contract_path(contract)
+    else:
+        path = _get_path_from_name(contract)
+
+    contract_class = compile_starknet_files(files=[path], debug_info=True)
+    return contract_class
+
```

Comment:
```
Traceback (most recent call last):
  File "/home/workshop/env/bin/nile", line 5, in <module>
    from nile.cli import cli
  File "/home/workshop/env/lib/python3.8/site-packages/nile/cli.py", line 20, in <module>
    from nile.utils.debug import debug as debug_command
  File "/home/workshop/env/lib/python3.8/site-packages/nile/utils/__init__.py", line 169, in <module>
    class Account:
  File "/home/workshop/env/lib/python3.8/site-packages/nile/utils/__init__.py", line 181, in Account
    get_class = get_contract_class("Account")
  File "/home/workshop/env/lib/python3.8/site-packages/nile/utils/__init__.py", line 136, in get_contract_class
    path = _get_path_from_name(contract)
  File "/home/workshop/env/lib/python3.8/site-packages/nile/utils/__init__.py", line 128, in _get_path_from_name
    raise FileNotFoundError(f"Cannot find '{name}'.")
FileNotFoundError: Cannot find 'Account'.
```
I think some of these functions weren't easily portable since they depend on structure and contents from Contracts. We might be better off leaving them out of this beta.